constants/csv_parser.ts

```diff
@@ -0,0 +1,224 @@
+import { Question, Answer } from '../models'
+import fs from 'fs'
+import path from 'path'
+import { parse } from 'csv-parse/sync'
+
+// The type containing columns that we care about in the Flowchart CSV
+type CsvType = {
+  Id: string
+  Name: string
+  'Line Source': string
+  'Line Destination': string
+  'Text Area 1': string
+  'Contained By': string
+}
+
+// Extracts the columns of note from the csv
+const csvToCsvTypeArray = (fileName: string): CsvType[] => {
+  const f = fs.readFileSync(path.resolve(__dirname, fileName))
+  const csv = parse(f, { columns: true }) as CsvType[]
+  return csv
+}
+
+// takes in a list of answers (not including tooltips) and outputs a map from the question id to that answer
+const mapAnswers = (answersArray: CsvType[]): Map<number, CsvType[]> => {
+  const map = new Map<number, CsvType[]>()
+  answersArray.forEach((answer: CsvType) => {
+    const existingAnswers = map.get(parseInt(answer['Line Source']))
+    if (existingAnswers) {
+      existingAnswers.push(answer)
+    } else {
+      map.set(parseInt(answer['Line Source']), [answer])
+    }
+  })
+  return map
+}
+
+// Transforms a csv entry into an Answer
+const answerFromCsvType = (answer: CsvType): Answer => {
+  if (
+    answer['Text Area 1'] === 'CONTINUE' ||
+    answer['Text Area 1'] === 'TEXT'
+  ) {
+    return {
+      route: parseInt(answer['Line Destination']),
+    }
+  } else {
+    return {
+      content: answer['Text Area 1'],
+      route: parseInt(answer['Line Destination']),
+    }
+  }
```

Comment:
this is a nit, so feel free to ignore if you'd like

seems like the if statement is only to add a single field, so i maybe do smth like

```suggestion
  const csvAnswer = { route: parseInt(answer['Line Destination']) };
  if (
    answer['Text Area 1'] !== 'CONTINUE' &&
    answer['Text Area 1'] !== 'TEXT'
  ) {
     csvAnswer['content'] = answer['Text Area 1'];
  }
  
  return csvAnswer;
```

Reply:
with the power of de morgan's law :hellmo:

alternatively, you could also have the if statement say 
```suggestion
  if (['CONTINUE' , 'TEXT'].includes(answer['Text Area 1'])) {
  ...
  }
```
but up to you, whatever seems readable 👍